README.rst

```diff
@@ -261,7 +261,8 @@ Dependencies
 
 A C compiler is required.  To use the Python API or the command-line tools a
 Python interpreter is also required.  We have tested it with Python v2.4,
-v2.5, v2.6, and v2.7.  For the Haskell interface, GHC >= 6.8.1 is required.
+v2.5, v2.6, v2.7, v3.3 and v3.5.  For the Haskell interface, GHC >= 6.8.1 is
```

Comment:
update: code in this PR can't work on 2.4 / 2.5 (see below).

and there is no reason to support these dead versions today, so just remove all references from docs etc.